sky/unit/test/widget/widget_tester.dart

```diff
@@ -117,11 +117,10 @@ class WidgetTester {
   void scroll(Widget widget, Offset offset, { int pointer: 1 }) {
     Point startLocation = getCenter(widget);
     Point endLocation = startLocation + offset;
-    HitTestResult result = _hitTest(startLocation);
     TestPointer p = new TestPointer(pointer);
-    _dispatchEvent(p.down(startLocation), result);
-    _dispatchEvent(p.move(endLocation), result);
-    _dispatchEvent(p.up(), result);
+    dispatchEvent(p.down(startLocation), startLocation);
+    dispatchEvent(p.move(endLocation), endLocation);
+    dispatchEvent(p.up(), endLocation);
```

Comment:
This change isn't correct.  In the real system, we don't perform a hit test for move and up.  We re-use the hit test that we got for down.  This change makes the test environment less like the real environment.